projects/dynamicforms/src/components/components.module.ts

```diff
@@ -44,23 +45,6 @@ import { CurrencyComponent } from './currency/currency.component';
     UploadComponent,
     CurrencyComponent,
   ],
-  entryComponents: [
-    AbstractFormComponent,
```

Comment:
Why we remove all entry components? Not needed anymore?

Reply:
Removed by mistake
Now it's restored

Reply:
Entry components are deprecated